gen/aircraft.py

```diff
@@ -265,6 +266,25 @@ def _start_type(start_type: str) -> StartType:
             return StartType.Cold
         return StartType.Warm
 
+    @staticmethod
+    def _start_type_at_group(
+        start_type: str,
+        unit_type: Type[FlyingType],
+        at: Union[ShipGroup, StaticGroup],
+    ) -> StartType:
+        group_units = at.dict().get("units")
```

Comment:
This should be `at.units`, guarded with an `isinstance` if needed. We shouldn't be dependent on the serialization format of the miz.